Sectors/Lines - LACC.txt

```diff
@@ -499,9 +499,9 @@ COORD:N051.29.60.000:E002.00.00.000
 COORD:N051.23.00.000:E001.59.60.000
 
 SECTORLINE:SAS1
-DISPLAY:London AC Clacton:London AC Clacton:EBBU
```

Comment:
I think this line should stay as EBBU or it should never turn on. The airspace here is a contiguous part of the UK FIR, from FL65 to FL660